### PR TITLE
grub: supply disk guid to randomize in coreos.randomize_disk_guid

### PR DESCRIPTION
**Pull request by vcaputo**

### CODE REVIEW

File: build_library/grub.cfg [hunk 1]
```diff
@@ -45,7 +45,7 @@ search --no-floppy --set first_boot \
 if [ -n "$first_boot" ]; then
     set first_boot="coreos.first_boot=1"
     # Explicitly request the disk-guid randomization
-    set randomize_disk_guid="coreos.randomize_disk_guid=1"
+    set randomize_disk_guid="coreos.randomize_disk_guid=00000000-0000-0000-0000-000000000001"
```
Comment (marineam) [inline build_library/grub.cfg]: one thought, we could reduce variable clutter by putting both of these options into the first_boot variable.

Comment (vcaputo) [inline build_library/grub.cfg]: i.e. coreos.first_boot.randomize_disk_guid=... ?

Comment (marineam) [inline build_library/grub.cfg]: No, `set first_boot="coreos.first_boot=1 coreos.randomize_disk_guid=00000000-0000-0000-0000-000000000001"`

Comment (vcaputo) [inline build_library/grub.cfg]: I sortof prefer treating the "randomize disk guid" and "first boot" as entirely independent things

Comment (marineam) [inline build_library/grub.cfg]: ok

Comment (vcaputo) [inline build_library/grub.cfg]: I see, you mean just variables within the script...

**Comment by marineam** [comment]
LGTM

**Comment by vcaputo** [comment]
combined vars into first_boot variable as requested, woot

**Pull request merged**